src/common/__tests__/merjoonTransformer.test.ts

```diff
@@ -68,6 +68,26 @@ describe('MerjoonTransformer', () => {
   });
 
   describe('parseValue', () => {
+    it('UUID', () => {
```

Comment:
do not it and describe on the same level, let's wrap with describes to keep things constant